src/Microsoft.ML.TimeSeries/STL/FastLoess.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// this is the fast version of Loess. there are several alternatives to improve the performance. this one is an approximation approach.
+    /// the smoothing is conducted on a sample set, and then the values on the left points are assigned directly.
+    /// </summary>
+    public class FastLoess
```

Comment:
This class is only used by `InnerStl`, please make it `internal` for the reasons explained here:
https://github.com/dotnet/machinelearning/pull/5202/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.csv#r440495855

Reply:
updated.

---
In reply to: [440505345](https://github.com/dotnet/machinelearning/pull/5202#discussion_r440505345) [](ancestors = 440505345)